packages/playground-ui/src/index.css

```
@@ -19,12 +19,12 @@
   /* Dark theme (default) */
-  --surface1: oklch(0.141 0 0);
-  --surface2: oklch(0.179 0 0);
-  --surface3: oklch(0.21 0 0);
-  --surface4: oklch(0.251 0 0);
-  --surface5: oklch(0.275 0 0 / 0.9);
-  --surface6: oklch(0.345 0 0);
+  --surface1: oklch(0 0 0);
+  --surface2: oklch(0.1684 0 0);
+  --surface3: oklch(0.1913 0 0);
+  --surface4: oklch(0.2178 0 0);
+  --surface5: oklch(0.2435 0 0 / 0.9);
```

Stylelint (17.11.0)

[error] Expected "0.9" to be "90%" (alpha-value-notation)

(alpha-value-notation)